### PR TITLE
Lesson 02 Assignments

### PR DESCRIPTION
**Pull request by kethsanders**

### CODE REVIEW

File: Lesson03/excercise-two.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+let sentence = "this has five words total.";
+
+let word = sentence.split(" ", 5);
```
Comment (jimmynono) [inline Lesson03/excercise-two.js]: Be careful with the limit number. It's not needed in this case, and might be problematic.

Also, use single quote marks in your JS files

```
let word = sentence.split(' ');
```

File: Lesson05/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <head>
+      <meta charset="utf-8">
+      <title>Banking Bankerson</title>
+      <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    </head>
+
+    <body>
+      <h1>Banking Bankerson</h1>
+      
+      <script src="./lesson5.js" charset="utf-8"></script>
```
Comment (jimmynono) [inline Lesson05/index.html]: nice job using the `/`

File: Lesson05/lesson5.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+let quit = false;
+let balance = 0;
+
+while (quit === false) {
+  let input = prompt('Please select one of the following options: Q to Quit; W to Withdraw funds; D to Deposit funds; B to display balance:');
+  let adjustedInput = input.toUpperCase();
+
+  if (adjustedInput === 'Q') {
```
Comment (jimmynono) [inline Lesson05/lesson5.js]: You could combine this if statement into the switch and just have the 'Q' as another case

File: Lesson05/lesson5.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  else {
+    switch (adjustedInput) {
+
+      case 'B': {
+        alert('Your balance is: ' + balance);
+        break;
+      }
+
+      case 'W': {
+        let withdraw = prompt('Enter amount to withdraw: ');
```
Comment (jimmynono) [inline Lesson05/lesson5.js]: You could simplify these statements like this...

```
let withdraw =Number(prompt('Enter amount to withdraw: '));
 balance -= adjustedWithdraw;
```

File: Lesson05/lesson5.js (hunk 3 above)
Comment (kethsanders) [reply Lesson05/lesson5.js]: I was having some trouble getting the switch to execute until I put into an if/else.

Will try again to see if I had just been encountering a code error (e.g. I typed something wrong).

**Pull request closed without merging**